dmi-id-dbus-service/README.md

```diff
@@ -0,0 +1,41 @@
+<!---
+  Copyright 2022 SECO Mind Srl
+
+  SPDX-License-Identifier: Apache-2.0
+-->
+
+# Hardware ID dbus Service Example
+Example project of a service that retrieves a hardware id from the BIOS (standard SMBIOS/DMI)
+and publishes it on a dbus channel.
```

Comment:
that either fetches hardware id from SMBIOS/DMI or from the kernel command line.